src/Resources/contao/library/Contao/DcaExtractor.php

```diff
@@ -401,6 +401,12 @@ protected function createExtract()
 			return;
 		}
 
+		// Return if the DC type is "Folder" and is not database assisted
+		if ($GLOBALS['TL_DCA'][$this->strTable]['config']['dataContainer'] == 'Folder' && empty($GLOBALS['TL_DCA'][$this->strTable]['config']['databaseAssisted']))
```

Comment:
I thought `dataContainer` is a required field because there is no isset check 6 lines above. Wouldn’t removing the isset check result in “undefined index” notices?